bot-api/pom.xml

```diff
@@ -56,6 +56,8 @@
                                     <http_proxy>${env.HTTP_PROXY}</http_proxy>
                                 </args>
                                 <from>python:3.8-slim-buster</from>
+                                <!-- network as host is useful when using proxy in localhost-->
+                                <!-- <network>host</network> -->
```

Comment:
Test if we can pass it as property outside this file, this will prevent from having this diff with the upstream repository.